### PR TITLE
Broadinstitute -> DataBiosphere changes

### PR DESCRIPTION
**Pull request by rushtong**
Addresses https://broadinstitute.atlassian.net/browse/GAWB-3024
Updates for the migration from broadinstitute -> DataBiosphere github/travis orgs.

See also: 
* https://github.com/DataBiosphere/consent/pull/249
* https://github.com/DataBiosphere/consent-ontology/pull/89

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Consent-UI
 ==========
 
-[![Build Status](https://travis-ci.com/broadinstitute/consent-ui.svg?token=3ve6QNemvC5zpJzsoKzf&branch=develop)](https://travis-ci.com/broadinstitute/consent-ui)
+[![Build Status](https://travis-ci.org/DataBiosphere/consent-ui.svg?branch=develop)](https://travis-ci.org/DataBiosphere/consent-ui)
```
Comment (davidangb) [inline README.md:4]: add the coverage badge? something like:
```
[![Coverage Status](https://coveralls.io/repos/DataBiosphere/consent-ui/badge.svg?branch=develop)](https://coveralls.io/r/DataBiosphere/consent-ui?branch=develop)
```

**Review by davidangb — COMMENTED** [review]
The PR shows that Travis passed, but the badge at https://github.com/DataBiosphere/consent-ui/tree/GAWB-3024 shows an error. Any idea why?

**Comment by rushtong** [comment]
I'm pretty sure that the badge visible on the repo home page is based off of the default branch, develop. The current develop travis build is, in fact, broken, due to differences between the travis node settings and what we use for the real build image node settings. My theory is that once a passing branch merges to develop, then the badge will reflect the new dev run.

**Review by davidangb — APPROVED** [review]
:+1: you're totally right the badge is pinned to the develop branch. All good!

**Review by aednichols — APPROVED** [review]

**Pull request merged**